lua/gitblame/init.lua

```diff
@@ -600,7 +597,7 @@ end
 
 M.copy_commit_url_to_clipboard = function()
     M.get_sha(function(sha)
-        if sha then
+        if sha and sha ~= "" then
```

Comment:
Could this be maybe extracted to an `is_sha_empty` function?

Reply:
Good idea!